llvm/test/CodeGen/PowerPC/peephole-replaceInstr-after-eliminate-extsw.mir

```diff
@@ -646,9 +648,10 @@ body:             |
     MTCTR8loop killed %139, implicit-def dead $ctr8
     B %bb.10
   
-  bb.8.for.cond21.for.cond.cleanup25_crit_edge:
+  bb.8.for.cond21.for.cond.cleanup25_crit_edge (landing-pad):
     successors: %bb.9(0x80000000)
-  
+    liveins: $x2
+
```

Comment:
`r2` in this case is used as toc pointer, IIUC, `r2` should be reserved and should not appear in liveins.